src/pages/settings/Profile/ProfilePage.js

```diff
@@ -213,120 +149,70 @@ class ProfilePage extends Component {
     }
 
     render() {
-        const pronounsList = _.map(this.props.translate('pronouns'), (value, key) => ({
-            label: value,
-            value: `${CONST.PRONOUNS.PREFIX}${key}`,
-        }));
         const currentUserDetails = this.props.currentUserPersonalDetails || {};
-        const pronounsPickerValue = this.state.hasSelfSelectedPronouns ? CONST.PRONOUNS.SELF_SELECT : this.pronouns;
-
+        const profileSettingsOptions = [
+            {
+                description: this.props.translate('displayNamePage.headerTitle'),
+                title: `${lodashGet(currentUserDetails, 'firstName', '')} ${lodashGet(currentUserDetails, 'lastName', '')}`,
```

Comment:
I like this, I thinkkkkk there's also `displayName` in `currentUserPersonalDetails`, right? Which do you think is better?

Reply:
Woah, I missed `displayName` in `currentUserPersonalDetails`. It is definitely better.

Reply:
fixed.